package.json

```diff
@@ -84,12 +83,12 @@
     "test:watch": "npm test -- --watch",
     "test:examples": "node examples/",
     "cover": "cross-env BABEL_ENV=commonjs istanbul cover _mocha -- --compilers js:babel-register --recursive",
-    "lint": "eslint src test --fix",
```

Comment:
Since `lint` was getting run on `prepare`/`prepublish`, the `--fix` flag was formatting the project w/o prettier -- which is how I discovered the discrepancy between the prettier config and formatting related eslint rules.